e2e/cypress/integration/ui-smoke/ui-launch-analysis.spec.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+import '../../support/commands';
+
+const resizeObserverLoopErrRe = /ResizeObserver loop limit exceeded/;
+const projectName = 'IntTest - Add Metadata Project';
+const gem2sTimeOut = (60 * 1000) * 30; // 30 minutes;
+const qcTimeOut = (60 * 1000) * 30; // 30 minutes;
+
+describe('Launches analysis successfully', () => {
+  // before each test:
+  //   1. set up the network intercepts
+  //   2. Log in into biomage
+  //   3. Visit data-management
+  beforeEach(() => {
+    // Intercept GET calls to */projects/* endpoint
+    cy.intercept(
+      {
+        method: 'GET',
+        url: '**/projects',
+      },
+    ).as('getProjects');
+
+    // Intercept GET calls to */experiments/* endpoint
+    cy.intercept(
+      {
+        method: 'GET',
+        url: '**/experiments',
+      },
+    ).as('getExperiment');
+
+    cy.login();
+    cy.visit('/data-management');
+  });
+
+  // we have some kind of resize observer loop error that needs looking into
+  Cypress.on('uncaught:exception', (err) => {
+    if (resizeObserverLoopErrRe.test(err.message)) {
+      return false;
+    }
+    return true;
+  });
```

Comment:
take a look whether this needs an actual fix

Reply:
This seems to happen because of our use of the `ReactResizeDetector` component in `data-management.jsx`. The component informs the element within it, the current size of the tile that component is rendered in. I tried running tests with this component removed and the error does not appear.

This error seems to becaused by React rerenders : https://github.com/maslianok/react-resize-detector/issues/45#issuecomment-858810843

I've created a separate ticket to address this : https://biomage.atlassian.net/browse/BIOMAGE-1406

Reply:
Can you add some details in that ticket, Agi?

Reply:
I'm sure I've added description inside the ticket :S Maybe I forgot to save it

Reply:
@ivababukova this error doesn't appear anymore after merging in your in the UI refactor, so I'm removing this. The test works in local. I'm testing it in staging.

Reply:
can you also mark this ticket you created as done then? And make sure it is not in a sprint

Reply:
The test completes in staging : https://github.com/biomage-ltd/testing/actions/runs/1220228086 so I'm removing this code and the ticket.